packages/http-event-normalizer/index.js

```diff
@@ -3,13 +3,6 @@ const httpEventNormalizerMiddleware = () => {
     const { event } = request
 
     const version = pickVersion(event)
-    const isHttpEvent = isVersionHttpEvent[version]?.(event)
-    if (!isHttpEvent) {
-      throw new Error('[http-event-normalizer] Unknown http event format')
-    }
-    // VPC Lattice is an http event, however uses a different notation
-    // - query_string_parameters
-    // - is_base64_encoded
```

Comment:
Can you leave this comment in please.

Reply:
my bad, didn't intend to remove that. I'll add it back